app/src/common/chromium/com/igalia/wolvic/browser/api/impl/PromptDelegateImpl.java

```diff
@@ -1011,56 +1016,62 @@ public boolean saveOrUpdatePassword(PasswordForm form) {
 
         @Override
         public boolean onLoginSelect(PasswordForm[] forms) {
-            dismiss();
-            assert mPasswordManagerListener != null;
-
-            WAutocomplete.LoginSelectOption[] options =
-                    new WAutocomplete.LoginSelectOption[forms.length];
-            for (int i = 0; i < forms.length; i++) {
-                options[i] = new WAutocomplete.LoginSelectOption(fromPasswordForm(forms[i]));
-            }
+            if (showLoginSelectDialog) {
```

Comment:
You can leave most of the code unchanged and the change more compact if you invert the condition
 ```
if (!showLoginSelectionDialog)
  return false;
```